app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -50,19 +50,22 @@
  */
 
 public class MainPlannerFragment extends Fragment implements BackPressedHandler, TripFragment.Callback,
-        DestinationFragment.Callback, DateFragment.Callback, LandmarkFragment.Callback{
-    public interface Callback {
-        void onLoadPage(PageTitle title, HistoryEntry entry);
-    }
-    @BindView(R.id.fragment_travel_planner_view_pager) ViewPager viewPager;
-    @BindView(R.id.planner_next) Button bNext;
-    @BindView(R.id.planner_save) Button bSave;
-    @BindView(R.id.planner_title) TextView tvTitle;
+        DestinationFragment.Callback, DateFragment.Callback, LandmarkFragment.Callback {
+
+    @BindView(R.id.fragment_travel_planner_view_pager)
+    ViewPager viewPager;
+    @BindView(R.id.planner_next)
+    Button bNext;
+    @BindView(R.id.planner_save)
+    Button bSave;
+    @BindView(R.id.planner_title)
+    TextView tvTitle;
 
     private PlannerFragmentPagerAdapter adapter;
     private Trip openTrip;
     private Unbinder unbinder;
-    private List<Trip> userTripsList = new ArrayList<>();
+    private List <Trip> userTripsList = new ArrayList <>();
+    private List <Trip> userDestinationList = new ArrayList <>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)